docker/entrypoint-local.sh

```diff
@@ -1,23 +1,5 @@
 #!/bin/bash
 
-mkdir -p \
-  /home/jovyan/.executor \
-  /home/jovyan/_jobs \
-  /home/jovyan/_jobs/auto-ml-demo/data \
-  /home/jovyan/_jobs/low-code-demo/data \
-  /home/jovyan/_jobs/executor-test/data
-
-cp -R /tmp/repo/data/notebooks /home/jovyan/.executor/notebooks
```

Comment:
I might be mistaken here, but shouldn't we be keeping the `executor` notebooks for the `local` environment? I believe we use these notebooks for the `papermill` execution, not just for the demos. Aren't we still going to allow the user to define a `study.yaml` file and run low-code/no-code notebooks via papermill?